index.html

```diff
@@ -0,0 +1,278 @@
+            </ul>
+          </div>
+        </header>
+        <section class="about-title" id="about-section">
+          <p class="sub-about">"Hello music world!"</p>
+          <h2 class="about-heading">New year music party at Giza pyramids</h2>
+          <p class="about-text">
+            The new year pyramids party brings up a various and marvilous
+            musicians, artists and singers. All at one of the most wonderful and
+            historical places of the world.You are all welcome to start a new
+            year with us at the most ancient state of the world.
+          </p>
+          <p class="booking">
+            Go a head and ask about the booking and any other details you need
+            to know. <br />
+            limited seats do not waste your chance!
+          </p>
+          <a href="#" class="booking-email">pyramidsparty2023@Mazzika.com</a>
+        </section>
+        <div class="logo">
+          <h2 class="logo-title">New year music party 2023</h2>
+          <p class="logo-text">
+            The logo of the party is a symbol of the pyramids of Giza.
+          </p>
+          <a href="#sub-heading1" class="logo-link">
+            <img
+              src="./assets/images/pyramid.svg"
+              alt="party-logo"
+              class="party-logo"
+            />
+          </a>
+        </div>
+        <section class="previous-parties">
+          <h2 class="previous-title">the previous parties</h2>
+          <p class="previous-text">Take a look at our last two parties.</p>
+          <div class="previous-photo">
+            <p class="party-2019">
+              <strong>2019</strong> <br />
+              <span>new year party</span>
+            </p>
+            <p class="party-2018">
+              <strong>2018</strong> <br />
+              <span>new year party</span>
+            </p>
+          </div>
+        </section>
+        <section class="sponsors">
+          <h2 class="sponsors-title">Our Sponsors</h2>
+          <ul class="sponsors-list">
+            <li>
+              <img
+                src="./assets/images/274409.svg"
+                alt="melody-logo"
+                class="melody-logo"
+              />
+            </li>
+            <li>
+              <img
+                src="./assets/images/Mazzika-01.svg"
+                alt="mazzika logo"
+                class="mazzika-logo"
+              />
+            </li>
+            <li>
+              <img
+                src="./assets/images/mbc-group-01.svg"
+                alt="mbc logo"
+                class="mazzika-logo"
+              />
+            </li>
+          </ul>
+        </section>
+        <footer class="about-footer">
+          <img
+            src="./assets/images/pyramid.svg"
+            alt="party-logo"
+            class="party-logo"
+          />
+          <p class="footer-text">2023 new year party at the pyramids</p>
+        </footer>
```

Comment:
- [ ] Both pages should contain the same navbar and footer. Kindly add footer in `index` page currently it is missing
- [ ] Please add partners section in index page as well:

Your design  | Behance design
------------- | -------------
<img width="768" alt="Screenshot 2022-04-14 at 16 31 50" src="https://user-images.githubusercontent.com/77038610/163391092-8418e33c-f436-4026-b4c8-c64cafcea95a.png"> | <img width="964" alt="Screenshot 2022-04-14 at 16 32 11" src="https://user-images.githubusercontent.com/77038610/163391167-dbe29a55-8e27-4c79-b3e9-e279ffef82d5.png">
 | Content Cell